client/src/utils/attributeAndParamUtils.js

```diff
@@ -409,12 +409,67 @@ const setOutputValueName = (activityId, value) => {
   );
 };
 
+/**
+ * @description If there is more than one unused parameter Object, delete it in the DB
+ * @param {Array} parameterObject List of all attributes saved in the sessionStorage
+ * @param {Array} usedIds The activityIds that are still being used
+ */
+const deleteUnusedParameterFromDB = (parameterObject, usedIds, robotId) => {
```

Comment:
robotId is missing in the documentation

Reply:
And the description of parameterObject should be updated to "parameterObject List of all **parameters** saved in the sessionStorage"